src/transformers/modeling_gguf_pytorch_utils.py

```diff
@@ -130,18 +130,6 @@ def load_gguf_checkpoint(gguf_checkpoint_path, return_tensors=False):
         if gguf_key in reader_keys:
             logger.info(f"Some keys were not parsed and added into account {gguf_key} | {value}")
 
-    # retrieve config vocab_size from tokenizer
-    # Pleas refer to https://github.com/huggingface/transformers/issues/32526 for more details
-    if "vocab_size" not in parsed_parameters["config"]:
-        tokenizer_parameters = parsed_parameters["tokenizer"]
-        if "tokens" in tokenizer_parameters:
-            parsed_parameters["config"]["vocab_size"] = len(tokenizer_parameters["tokens"])
-        else:
-            logger.warning(
-                "Can't find a way to retrieve missing config vocab_size from tokenizer parameters. "
-                "This will use default value from model config class and cause unexpected behavior."
-            )
-
```

Comment:
is this fixed in the new version of gguf ?

Reply:
Ooops, this is a mistake in rebase. Reverted.